app/core/utils.js

```diff
@@ -196,6 +196,21 @@ utils.getInviteKey = function(location) {
   return '';
 }
 
+utils.getRoles = function(location) {
```

Comment:
We've got tests around other things in this utils module. Could you add some tests around this @darinkrauss? They are in test/unit/utils/utils.test.js it looks like. Will also be helpful as "documentation" since there's a lot going on in your lodash filtering!